railties/test/application/test_runner_test.rb

```diff
@@ -445,7 +445,8 @@ def test_raise_error_when_specified_file_does_not_exist
     def test_pass_TEST_env_on_rake_test
       create_test_file :models, 'account'
       create_test_file :models, 'post', pass: false
-
+      # This specifically verifies TEST for backwards compatibility with rake test
+      # as bin/rails test already supports running tests from a single file more cleanly.
```

Comment:
Personally don't think this adds much. The fact that it's run with `bin/rake` should be enough of a clue. But I'll let the build run and if it's green, I'll merge 👍